### PR TITLE
[zh-cn] Blog: localization 2022-12-23-fsgroup-on-mount.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn] Blog: localization 2022-12-23-fsgroup-on-mount.md

### CODE REVIEW

**Comment by mengjiao-liu** [comment]
Preview page: https://deploy-preview-39186--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/12/23/kubernetes-12-06-fsgroup-on-mount/

File: content/zh-cn/blog/_posts/2022-12-23-fsgroup-on-mount.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+<!--
+## Why is it important?
+
+Without this feature, applying the fsGroup information to files is not possible in certain storage environments.
+
+For instance, Azure File does not support a concept of POSIX-style ownership and permissions
+of files. The CSI driver is only able to set the file permissions at the volume level.
+-->
+## 它为什么如此重要？
```
Comment (tengqm) [inline content/zh-cn/blog/_posts/2022-12-23-fsgroup-on-mount.md:115]:
```suggestion
## 这一特性为何重要？
```

**Review by tengqm — COMMENTED** [review]
Overall Looks Good!
/approve

Some nits inline.

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Pull request merged**